api/rest/bundle_handler.go

```diff
@@ -72,6 +73,12 @@ type BundleHandler struct {
 	bundleCreationTimeout time.Duration         // limits how long bundle creation could take
 }
 
+type node struct {
```

Comment:
It was originally here since it's part of the input arguments for the bundlehandler creation endpoint so I decided to put it here now rather than put it in coordinator then move it in the next PR.